aws-athena/views/default-vw_pin_universe.sql

```diff
@@ -1,4 +1,29 @@
 -- Source of truth view for PIN location
+
+-- This CTE ensures the most recent year of pardat data is joined to spatial
+-- data regardless of timing. This is necessary since spatial data releases late
+-- in a calendar year while the CCAO's universe of PINs is rolled-over much
+-- sooner.
+WITH pardat_adjusted_years AS (
+
+    SELECT
+        parid,
+        taxyr,
+        CASE
+            WHEN
+                taxyr
+                > (SELECT MAX(year) FROM {{ source('spatial', 'parcel') }})
+                THEN (SELECT MAX(year) FROM {{ source('spatial', 'parcel') }})
+            ELSE taxyr
+        END AS join_year,
+        class,
+        nbhd,
+        cur,
+        deactivat
+    FROM {{ source('iasworld', 'pardat') }}
+
+)
+
```

Comment:
**question:** Worth making into its own ephemeral model or view?

Reply:
Our naming syntax for symlinks in dbt/models/ makes ethereal tables error out. Probably not worth working around ATM.